sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts

```diff
@@ -21,12 +21,14 @@ class MissingClientIdError extends Error {
 
 enum GapiScopes {
   BIGQUERY,
+  CLOUD,
   DRIVE,
   GCS,
 }
 
-// Authorize GCP scope by default
-const initialScopeString = 'profile email https://www.googleapis.com/auth/cloud-platform';
+// Ask for all necesary scopes up front so we don't need to ask again later.
+// Cloud-platform scope covers BigQuery and GCS but not Drive.
+const initialScopes = [ GapiScopes.CLOUD, GapiScopes.DRIVE ];
```

Comment:
Do we need to include "profile" and "email" here?

Reply:
Yeah, I just noticed that, will put those back in.

Reply:
OK, done.